src/writer.jl

```diff
@@ -167,9 +167,18 @@ function write(path::AbstractString, obj; force=false)
     Base.write(paths.shx, index_handle)
 
     # Write .prj file
-    if obj isa Shapefile.Table
-        crs = GeoInterface.crs(obj)
-        crs isa GeoFormatTypes.ESRIWellKnownText && Base.write(paths.prj, crs.val)
+    crs = try
+        GI.crs(geoms)
```

Comment:
I think on getting the crs from the `obj` might be more valid than on the vector of geometries. This is how I now do it in https://github.com/evetion/GeoDataFrames.jl/blob/master/src/utils.jl#L19

Reply:
Done!